coa-helm/values.yaml

```diff
@@ -12,6 +12,7 @@
   dockerInDockerImageName: ""
   usePreemptibleVmsOnly: false
   drsUrl: RUNTIME_PARAMETER
+  workflowExecutionIdentity: RUNTIME_PARAMETER

```

Comment:
FWIW, I _think_ this is the same value as `identity.resourceId` (L37) which Leo is already setting -- but updating Leo to send `config.workflowExecutionIdentity` as well.